sdk/search/Azure.Search.Documents/src/Generated/Models/AnalyzeResult.Serialization.cs

```diff
@@ -14,19 +15,21 @@ public partial class AnalyzeResult
     {
         internal static AnalyzeResult DeserializeAnalyzeResult(JsonElement element)
         {
-            AnalyzeResult result = new AnalyzeResult();
+            IReadOnlyList<TokenInfo> tokens = new List<TokenInfo>();
             foreach (var property in element.EnumerateObject())
             {
                 if (property.NameEquals("tokens"))
                 {
+                    List<TokenInfo> array = new List<TokenInfo>();
                     foreach (var item in property.Value.EnumerateArray())
                     {
-                        result.Tokens.Add(TokenInfo.DeserializeTokenInfo(item));
+                        array.Add(TokenInfo.DeserializeTokenInfo(item));
```

Comment:
Nit: I prefer splitting these up for easier debugging, i.e. easier to see what `DeserializedTokenInfo` returned, then can put it in the `array`. Combined, it's more work to examine what was just added.